trojsten/submit/views.py

```diff
@@ -238,6 +241,21 @@ def poll_submit_info(request, submit_id):
     }), content_type='application/json; charset=utf-8')
 
 
+def send_notification_email(submit):
+    send_mail(
+        _('[Trojstenweb] User submission detected'),
+        _('{name} submitted {possessive} solution to task {task}\n\nSubmit link: {submit_link}\n\nThis is an automated'
```

Comment:
Toto chce preklad - https://docs.djangoproject.com/en/2.0/topics/i18n/translation/
Chces v podstaten napisat preklad do `trojsten/locale/sk/LC_MESSAGES/django.po`
a spustiť `compilemessages`.

Otestovať si to vieš tak, že submitneš a v development omode sa ti mail miesto poslania vypíše na konzolu.